app/api/grape_api/user_api.rb

```diff
@@ -0,0 +1,191 @@
+module GrapeAPI
```

Comment:
@mssola the functionality slightly different. API can create tokens for any user, not only for current, and return plain_token. I think changing the frontend to use the API is preferred.
Will add more doc.